src/components/common/Directors.vue

```diff
@@ -61,19 +61,19 @@
                     <v-text-field filled class="item"
                       label="First Name"
                       id="new-director__first-name"
-                      v-model="director.officer.firstName"
```

Comment:
There were too many things named "director" or "directors" !!!

The new name identifies this variable's specific purpose.